test/test_webdriver_utils.py

```diff
@@ -26,10 +26,11 @@ def test_parse_neterror_integration(self):
         manager.close()
         # I couldn't figure out how to just get a the get_command
         # Because the type has too many quotes
```

Comment:
This one too

Reply:
No, the two lines should be removed.
Btw you can do multiline comments by clicking and dragging on the left side of the screen.

Reply:
Ah... Didn't know that😅, good to know.